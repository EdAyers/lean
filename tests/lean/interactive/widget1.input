{"seq_num": 0, "command": "sync", "file_name": "widget1.lean"}
{"seq_num": 46, "command": "info", "file_name": "widget1.lean", "line": 6, "column": 12}
{"seq_num": 47, "command" : "widget_event", "handler" : 0, "route" : [22, 3], "file_name": "widget1.lean", "line": 6, "column": 12, "args" : { "type" : "string", "value" : "hello world"}}
{"seq_num": 48, "command" : "widget_event", "handler" : 3, "route" : [22, 3], "file_name": "widget1.lean", "line": 6, "column": 12, "args" : { "type" : "unit", "value" : {}}}
{"seq_num": 49, "command" : "widget_event", "handler" : 6, "route" : [3], "file_name": "widget1.lean", "line": 6, "column": 12, "args" : { "type" : "unit", "value" : {}}}
