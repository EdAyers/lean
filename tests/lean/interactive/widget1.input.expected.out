{"message":"file invalidated","response":"ok","seq_num":0}
{"record":{"doc":"Repeatedly apply `intro1` and return the list of new local constants in order of introduction.","full-id":"tactic.intros","source":,"tactic_params":[],"type":"tactic (list expr)","widget":{"html":[{"attributes":null,"children":[],"tag":"ol"},[{"attributes":{"type":"text","value":""},"children":[],"events":{"onChange":{"handler":0,"route":[22,3]}},"tag":"input"},{"attributes":null,"children":["+"],"events":{"onClick":{"handler":1,"route":[22,3]}},"tag":"button"}]]}},"response":"ok","seq_num":46}
{"record":{"debug":{"column":2,"line":6,"msg":"from m_goal_pos"},"status":"success","widget":{"html":[{"attributes":null,"children":[],"tag":"ol"},[{"attributes":{"type":"text","value":"hello world"},"children":[],"events":{"onChange":{"handler":2,"route":[22,3]}},"tag":"input"},{"attributes":null,"children":["+"],"events":{"onClick":{"handler":3,"route":[22,3]}},"tag":"button"}]]}},"response":"ok","seq_num":47}
{"record":{"debug":{"column":2,"line":6,"msg":"from m_goal_pos"},"status":"success","widget":{"html":[{"attributes":null,"children":[{"attributes":{"key":"0"},"children":["hello world",{"attributes":null,"children":["X"],"events":{"onClick":{"handler":6,"route":[3]}},"tag":"button"}],"tag":"li"}],"tag":"ol"},[{"attributes":{"type":"text","value":""},"children":[],"events":{"onChange":{"handler":7,"route":[23,3]}},"tag":"input"},{"attributes":null,"children":["+"],"events":{"onClick":{"handler":8,"route":[23,3]}},"tag":"button"}]]}},"response":"ok","seq_num":48}
{"record":{"debug":{"column":2,"line":6,"msg":"from m_goal_pos"},"status":"success","widget":{"html":[{"attributes":null,"children":[],"tag":"ol"},[{"attributes":{"type":"text","value":""},"children":[],"events":{"onChange":{"handler":9,"route":[24,3]}},"tag":"input"},{"attributes":null,"children":["+"],"events":{"onClick":{"handler":10,"route":[24,3]}},"tag":"button"}]]}},"response":"ok","seq_num":49}
